packages/vm/src/wasm_backend/gatekeeper.rs

```diff
@@ -656,7 +673,6 @@ impl FunctionMiddleware for FunctionGatekeeper {
             | Operator::Catch { .. }
             | Operator::Throw { .. }
             | Operator::Rethrow { .. }
-            | Operator::Unwind { .. }
```

Comment:
For the record, this was removed here: https://github.com/bytecodealliance/wasm-tools/pull/288